lib/dev3/web/controllers/api/slack/slash_commands_controller.ex

```diff
@@ -63,18 +65,22 @@ defmodule Dev3.Web.API.Slack.SlashCommandsController do
   defp parse_args(%{params: %{"text" => args, "command" => command}} = conn, _) do
     case parse(args) do
       {:ok, parsed_args} -> assign(conn, :args, parsed_args)
-      :no_args_error -> handle_no_args_error(conn, command)
+      :no_args_error     -> handle_no_args_error(conn, command)
     end
   end
   defp parse_args(conn, _) do
     conn |> send_resp(:ok, "Args parsing error") |> halt
   end
 
   defp parse(args) when is_nil(args) or args == "", do: :no_args_error
-  defp parse(args), do: {:ok, String.split(args, " ")}
+  defp parse(args), do: {:ok, String.split(args, " ") |> limit_repos}
 
   defp handle_no_args_error(conn, command) do
     @slack_messenger.notify(:no_args_response, conn.assigns[:user], command)
     conn |> send_resp(:ok, "Args parsing error") |> halt
   end
+
+  defp limit_repos(parsed_arg) do
+    parsed_arg |> Enum.take(5)
```

Comment:
Better put the number of repos kept in a config var and use it here instead of hardcoding the number :) You can add it under the `rate_limiter` config (`max_repos_per_command` is an example).